Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM quay.io/3scale/ruby:2.1
+MAINTAINER Oriol Martí <oriol@3scale.net> # 2015-06-26
+
+RUN apt-install -y libssl-dev
+
+WORKDIR /opt/echo-api
+
+COPY Gemfile /opt/echo-api/
+COPY Gemfile.lock /opt/echo-api/
+
+RUN chown -fR ruby:ruby /opt/echo-api
+USER ruby
+
+RUN bundle install --deployment --without development test
+
+COPY config /opt/echo-api/config
```

Comment:
This can be just `COPY . /opt/echo-api/` right?

And because there is `WORKDIR` already, it can be just `COPY . ./`. All the paths can be relative.